src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -53,6 +53,12 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
     const { profile, symbolicationStatus, resymbolicateProfile } = this.props;
     const isSymbolicated = profile.meta.symbolicated;
 
+    const isSymbolicationUnsupported =
+      profile.meta.symbolicationNotSupported === false;
```

Comment:
`symbolicationNotSupported == false` doesn't sound logically the thing we want. It should be `symbolicationNotSupported === true` instead for your case and print `true` in your converter, right?
Currently `!(symbolication not supported)` == `symbolication supported`.

Other option is to have `symbolicationSupported === false` but I think "NotSupported" is better since our default is to support symbolication.

Reply:
You're right